### PR TITLE
sigstore, test: reflow, fix

### PR DESCRIPTION
**Pull request by woodruffw**
Fixes an online OIDC test regression.

### CODE REVIEW

**Review by tnytown — DISMISSED** [review]

File: test/unit/test_sign.py [hunk 1]
```diff
@@ -75,10 +75,12 @@ def test_sct_verify_keyring_lookup_error(signer, monkeypatch):
 
     with pytest.raises(
         InvalidSCTError,
-        match="Invalid key ID in SCT: not found in current keyring",
-    ):
+    ) as excinfo:
         signer.sign(payload, token)
 
+    # The exception subclass is the one we expect.
+    assert isinstance(excinfo.value, InvalidSCTKeyError)
```
Comment (tnytown) [inline test/unit/test_sign.py:82]: can we check for this specific subclass with `pytest.raises`?

Comment (woodruffw) [reply test/unit/test_sign.py:82]: We can, but I actually left it that way on purpose -- checking for the parent class being raised means that we haven't broken our exception contract around what this API raises. The `isinstance` check is then just a stand-in for ensuring that we got the right sub-error 🙂

**Review by tnytown — APPROVED** [review]

File: test/unit/test_sign.py [hunk 2]
```diff
@@ -75,10 +75,12 @@ def test_sct_verify_keyring_lookup_error(signer, monkeypatch):
 
     with pytest.raises(
         InvalidSCTError,
-        match="Invalid key ID in SCT: not found in current keyring",
```
Comment (di) [inline test/unit/test_sign.py:78]: Aren't we losing a bit of our test by removing this?

Comment (woodruffw) [reply test/unit/test_sign.py:78]: > Aren't we losing a bit of our test by removing this?

That's why I added the instance check that follows it -- we're checking that we don't change the top-level exception type with this `pytest.raises`, and then the following check makes sure that the specific exception type is the one reflecting a missing key ID.

(The `match` string no longer exists -- it's baked into the diagnostics now.)

**Review by di — APPROVED** [review]

**Pull request merged**